lib/fastly/dictionary_item.rb

```diff
@@ -0,0 +1,35 @@
+class Fastly
+  class DictionaryItem < Base
+    attr_accessor :dictionary_id, :item_key, :item_value, :service_id
+
+    # Return the Service object this belongs to
+    def service
+      @service ||= fetcher.get(Service, service_id)
+    end
+
+    # :nodoc:
+    def as_hash
+      super.delete_if { |var| %w(service_id dictionary_id).include?(var) }
+    end
+
+    def self.get_path(service, dictionary_id, item_key, _opts = {})
+      "/service/#{service}/dictionary/#{dictionary_id}/item/#{item_key}"
+    end
+
+    def self.post_path(opts)
+      "/service/#{opts[:service_id]}/dictionary/#{opts[:dictionary_id]}/item"
+    end
+
+    def self.put_path(opts)
+      get_path(opts[:service_id], opts[:dictionary_id], opts[:item_key])
```

Comment:
I believe we want to use the object here instead of a hash so that the `delete!` and `save!` methods in the [Base class](https://github.com/fastly/fastly-ruby/blob/master/lib/fastly/base.rb#L18) work. i.e.

```
def self.put_path(obj)
  get_path(obj.service_id, obj.dictionary_id, obj.item_key)
end
```